app/views/shared/_footer.html.erb

```diff
@@ -60,23 +60,23 @@
 
         <ol class="unstyled-list l-footer-social__list">
           <li class="l-footer-social__list-item twitter">
-            <a href="http://twitter.com/YourMoneyAdvice" title="Your Money Advice on Twitter">
+            <a href="http://twitter.com/YourMoneyAdvice" title="Your Money Advice on Twitter" class="footer-social__link">
```

Comment:
I think it should definitely be consistent with the rest of the SCSS file. It might be worth adding a new SCSS partial for these social links (with the other `l-footer-social` styles, aI agree it's not a layout component)